Delver/Assets/Source/Gameplay/Abilities/Ability_SimpleMelee.cs

```diff
@@ -33,11 +32,7 @@ public override void UpdateAbility(Actor abilityUser)
 
         if(currentCollider != null)
         {
-            if(!stopMovement && (currentCollider.transform.position - activatedPosition).sqrMagnitude < (maxRange * maxRange))
-            {
-                currentCollider.transform.position += activatedDirection * colliderSpeed * Time.deltaTime;
-            }
-            else
+            if (stopMovement && (currentCollider.transform.position - activatedPosition).sqrMagnitude < (maxRange * maxRange))
```

Comment:
hmm actually now that I look at it, I think it should be 
stopMovement || (currentCollider.transform.position - activatedPosition).sqrMag > (maxRange squared)

Cause we want it to stop movement when its forced to stop, or if it moves far enough away from the activated position. Right now the second check is backwards cause its true while you're NOT past the max range.